app/app.html

```diff
@@ -2,9 +2,15 @@
 <html>
   <head>
     <meta charset="utf-8">
+    <meta http-equiv="Content-Security-Policy" content="
+      default-src 'self';
+      connect-src 'self' http://localhost:* ws://localhost:* https://api.coinmarketcap.com http://zap.jackmallers.com https://testnet-api.smartbit.com.au;
+      script-src 'self' http://localhost:* 'unsafe-eval' 'unsafe-inline';
+      font-src 'self' data: http://localhost:* https://fonts.googleapis.com https://s3.amazonaws.com https://fonts.gstatic.com;
```

Comment:
https://s3.amazonaws.com is pretty broad, how about narrowing that down?

Reply:
I don't know that it is possible to be more specific than the host level in a CSP.

See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/font-src

Reply:
Fair enough then we would have to expose s3 assets via our own url. This works as an improvement without that.